src/server/nickel.rs

```diff
@@ -2,13 +2,16 @@
 //! (optional, enables `hyper` feature).
 //!
 //! Not shown here: [`impl HttpRequest for &mut nickel::Request`](../trait.HttpRequest.html#implementors).
-
+#![deprecated(since = "0.10.2", note = "Nickel integration has moved to the `multipart-nickel`
+                                        crate; in-crate integration will be removed in 0.11.0")]
 use nickel::Request as NickelRequest;
 
 use hyper::server::Request as HyperRequest;
 
 use super::HttpRequest;
 
+#[deprecated(since = "0.10.2", note = "Nickel integration has moved to the `multipart-nickel`
```

Comment:
Unfortunately, something tells me this won't actually warn at the use site. I may want to add a `warn!()` in here so it at least shows up in logs.